Gopkg.lock

```diff
@@ -311,41 +245,35 @@
   branch = "v2"
   name = "gopkg.in/yaml.v2"
   packages = ["."]
-  revision = "eb3733d160e74a9c7e442f435eb3bea458e1d19f"
+  revision = "d670f9405373e636a5a2765eea47fac0c9bc91a4"
 
 [[projects]]
-  branch = "release-1.8"
+  branch = "master"
```

Comment:
You're going to want to pin these to the branch version.

Reply:
Then call dep-ensure.

Reply:
ahh ok, let me try agian